pages/index.js

```diff
@@ -1,9 +1,96 @@
 import Head from "next/head";
+import styled from "styled-components";
+import RegisterButton from "../components/RegisterButton";
+import RegisterForm from "../components/RegisterForm";
+import {useState} from "react";
 
 export default function Home() {
+  const [register, setRegister] = useState(false);
+  const [loginMode, setLoginMode] = useState("parent");
+
+  const handleClickRegister = () => {
+    setRegister(true);
+  };
   return (
-    <Head>
-      <title>kidsFi - Finance for kids</title>
-    </Head>
+    <>
+      <Head>
+        <title>kidsFi - Finance for kids</title>
+      </Head>
+
+      <MainSection>
+        <Heading>kidsFi - Finance for kids</Heading>
+        <StyledParagraph>
+          The app every child needs to have a visual overview of their money.
+        </StyledParagraph>
+        <FlexContainer>
+          <StyledDiv>
+            <p>
+              Don&apos;t take our word for it though, please register and try it
+              out now.
+            </p>
+            <RegisterButton
+              register={register}
+              handleClickRegister={handleClickRegister}
+            />
```

Comment:
Please follow the convention of naming the props `on...` and only the handler `handle...`. So this line should read `onClickRegister={handleClickRegister}` (you have to adjust the prop name in your component as well)